<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visualisierung 1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="labelTop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>1001</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No data loaded</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="textVisible">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>771</width>
      <height>391</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="value" stdset="0">
     <number>0</number>
    </property>
    <property name="textVisible" stdset="0">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="CCW_rotation">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>530</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="ccw">
      <normaloff>CCW.png</normaloff>
      <normalon>CCW.png</normalon>CCW.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CW_rotation">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>580</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset theme="cw">
      <normaloff>CW.png</normaloff>
      <normalon>CW.png</normalon>CW.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QSlider" name="color_bar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>751</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QRadioButton" name="MIP_trans">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>530</y>
      <width>301</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>MIP </string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="FirstHit_trans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>311</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>First Hit </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="alpha_trans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>590</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Alpha Compositing </string>
    </property>
   </widget>
   <widget class="QRadioButton" name="Grad_trans">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>620</y>
      <width>381</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Diffuses Gradientenshading</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMinusZ">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Bottom</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPlusY">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPlusX">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>610</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Front</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushPlusZ">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>530</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Top</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMinusY">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>570</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Right</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushMinusX">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>610</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>View Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>769</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open ...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
